lib/twitter_cldr/js/renderers/numbers/rbnf/rbnf_test_helper_renderer.rb

```diff
@@ -0,0 +1,41 @@
+# encoding: UTF-8
+
+# Copyright 2012 Twitter, Inc
+# http://www.apache.org/licenses/LICENSE-2.0
+
+module TwitterCldr
+  module Js
+    module Renderers
+      module TestHelpers
```

Comment:
Module nesting should match directory structure, so this file should either be moved under `renderers/test_helpers` or the class moved from `TestHelpers` module into `Numbers`.